packages/wallets/injective/src/okx.ts

```diff
@@ -0,0 +1,32 @@
+import { Wallet as WalletType } from "@injectivelabs/wallet-ts";
+import { WalletState } from "@xlabs-libs/wallet-aggregator-core";
+import { InjectiveWallet } from "./injective";
+import { ConcreteWalletConfig } from "./types";
+
+export class OKXWallet extends InjectiveWallet {
+  constructor(config: ConcreteWalletConfig) {
+    super({
+      ...config,
+      type: "keplr" as WalletType,
```

Comment:
Does the OKX wallet set the `window.keplr` object?

Reply:
yes, okx wallet set window.keplr